packages/@expo/cli/src/start/index.ts

```diff
@@ -77,6 +78,7 @@ export const expoStart: Command = async (argv) => {
 
     --dev-client                           Experimental: Starts the bundler for use with the expo-development-client
     --force-manifest-type <manifest-type>  Override auto detection of manifest type
+    --private-key-path <path>              Path to private key for code signing. Default: "private-key.pem" in the same directory as the certificate specified by the expo-updates configuration in app.json.
```

Comment:
The start command does a lot of stuff, maybe `signing-key` would be a more useful name? This would be akin to `serve` CLI which uses `--ssl-key`.
```suggestion
    --signing-key <path>              Path to private key for code signing. Default: "private-key.pem" in the same directory as the certificate specified by the expo-updates configuration in app.json.
```

Reply:
`eas-cli` uses `--private-key-path` so I'm tempted to keep it for consistency. That being said I don't feel too strongly about naming so I'm happy to change this if you think the name is clearer and the benefit outweighs the cost of inconsistency (I guess we could rename the eas-cli one as well though.

Reply:
I do like the consistency aspect so I'm not in love with changing the arg. 

I just checked the help of `eas update` and noticed a bit of a naming inconsistency there as well which leads me to think it doesn't matter much (dir, path)
```
--input-dir=<value>
--private-key-path=<value> 
```

Personally I like the shorter argument names because I'm lazy.